[uWSGI] getting INI configuration from /home_daint/bbpnrsoa/.local/etc/nginx/uwsgi-nrp.ini
*** Starting uWSGI 2.0.18-debian (64bit) on [Fri Sep 24 13:00:46 2021] ***
compiled with version: 10.0.1 20200405 (experimental) [master revision 0be9efad938:fcb98e4978a:705510a708d3642c9c962beb663c476167e4e8a4] on 11 April 2020 11:15:55
os: Linux-4.12.14-197.7_5.0.100-cray_ari_c #1 SMP Fri Jul 23 15:32:31 UTC 2021 (c47993f)
nodename: nid01308
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 72
current working directory: /
detected binary path: /usr/bin/uwsgi-core
chdir() to /home_daint/bbpnrsoa/nrp/src/ExDBackend/hbp_nrp_backend/
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 514025
your memory page size is 4096 bytes
detected max file descriptor number: 262140
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/nrp-services.sock fd 3
Python version: 3.8.5 (default, Jan 27 2021, 15:41:15)  [GCC 9.3.0]
PEP 405 virtualenv detected: /home_daint/bbpnrsoa/.opt/platform_venv
Set PythonHome to /home_daint/bbpnrsoa/.opt/platform_venv
Python main interpreter initialized at 0x55555567ed30
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 146112 bytes (142 KB) for 8 cores
*** Operational MODE: threaded ***
/home_daint/bbpnrsoa/nrp/src/ExDBackend/hbp-flask-restful-swagger-master/flask_restful_swagger/swagger.py:322: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if op.get(att_name) and att_name is not 'nickname':
2021-09-24 13:00:48,125 [MainThread  ] [hbp_nrp_back] [WARNING]  Application started with uWSGI or any other framework. logging to console by default !
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x55555567ed30 pid: 287 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 287, cores: 8)
2021-09-24 13:01:48,270 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Could not generate textures, error occurred : Failed to communicate with the storage server, status code 500
2021-09-24 13:01:48,473 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Requesting simulation resources
2021-09-24 13:01:48,473 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation timeout initialized
2021-09-24 13:02:13,218 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/reset
2021-09-24 13:02:13,222 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/extend_timeout
2021-09-24 13:02:13,226 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/get_transfer_functions
2021-09-24 13:02:13,230 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/add_transfer_function
2021-09-24 13:02:13,234 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/activate_transfer_function
2021-09-24 13:02:13,238 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/edit_transfer_function
2021-09-24 13:02:13,242 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/convert_transfer_function_raw_to_structured
2021-09-24 13:02:13,245 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/delete_transfer_function
2021-09-24 13:02:13,249 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/get_brain
2021-09-24 13:02:13,253 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/set_brain
2021-09-24 13:02:13,257 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/get_populations
2021-09-24 13:02:13,261 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/set_populations
2021-09-24 13:02:13,265 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/get_CSV_recorders_files
2021-09-24 13:02:13,269 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/simulation_recorder
2021-09-24 13:02:13,273 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/get_robots
2021-09-24 13:02:13,276 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/add_robot
2021-09-24 13:02:13,280 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/del_robot
2021-09-24 13:02:13,284 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/set_exc_robot_pose
2021-09-24 13:02:13,287 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/0/prepare_model
2021-09-24 13:02:13,291 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation initialized
2021-09-24 13:02:14,366 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Starting State Machines...
2021-09-24 13:04:29,212 [/ros_cle_sim] [hbp_nrp_back] [INFO]  State machine outcomes: 
[WARN] [1632481469.214048, 1.000000]: Could not process inbound connection: [/nrp_backend] is not a publisher of [/ros_cle_simulation/cle_error]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/ros_cle_simulation/0/lifecycle', 'cle_ros_msgs/SimulationLifecycleStateChange']]{'callerid': '/rosbridge_websocket', 'md5sum': 'a2eeba76fe73c5cc85d54c7b17af3ec8', 'message_definition': '# This message contains information on a error raised after a user update of some CLE related source code:\n# Transfer function, Smach script, PyNN script\nint32 severity # error severity according to the following constants\nint32 SEVERITY_WARNING=0  # The error might not affect the simulation at all\nint32 SEVERITY_ERROR=1    # The error will lead to a simulation failure if not resolved\nint32 SEVERITY_CRITICAL=2 # The error has lead to simulation failure\nint32 SEVERITY_MAJOR = 3 # The error has lead to stop the simulation and show a message in the frontend\n\nstring sourceType # e.g., "Transfer Function"\nstring SOURCE_TYPE_TRANSFER_FUNCTION=Transfer Function\nstring SOURCE_TYPE_STATE_MACHINE=State Machine\nstring errorType # e.g., "NoOrMultipleNames", "Compile", "Loading", "Runtime"\nstring message # description of the error, e.g., "IndentationError: unexpected indent"\n\nstring functionName # python def name of the function causing the error, empty if unavailable\n\n# the following fields are used when a python SyntaxError is raised, strings are left empty otherwise\nint32 lineNumber # line number of the error, -1 if unavailable \nint32 offset # python\'s SyntaxError offset, -1 if unavailable\nstring lineText # text of the line causing the error\nstring fileName # name of the file in which the error is raised (<string> if no such file exists)\n', 'tcp_nodelay': '0', 'topic': '/ros_cle_simulation/cle_error', 'type': 'cle_ros_msgs/CLEError'}
2021-09-24 13:05:48,209 [Thread-5    ] [hbp_nrp_back] [INFO]  Start cleanup
2021-09-24 13:05:50,436 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Could not generate textures, error occurred : Failed to communicate with the storage server, status code 500
2021-09-24 13:05:50,679 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Requesting simulation resources
2021-09-24 13:05:50,679 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation timeout initialized
2021-09-24 13:06:05,031 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/reset
2021-09-24 13:06:05,036 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/extend_timeout
2021-09-24 13:06:05,040 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/get_transfer_functions
2021-09-24 13:06:05,044 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/add_transfer_function
2021-09-24 13:06:05,048 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/activate_transfer_function
2021-09-24 13:06:05,052 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/edit_transfer_function
2021-09-24 13:06:05,056 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/convert_transfer_function_raw_to_structured
2021-09-24 13:06:05,060 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/delete_transfer_function
2021-09-24 13:06:05,064 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/get_brain
2021-09-24 13:06:05,069 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/set_brain
2021-09-24 13:06:05,072 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/get_populations
2021-09-24 13:06:05,076 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/set_populations
2021-09-24 13:06:05,080 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/get_CSV_recorders_files
2021-09-24 13:06:05,084 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/simulation_recorder
2021-09-24 13:06:05,088 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/get_robots
2021-09-24 13:06:05,092 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/add_robot
2021-09-24 13:06:05,095 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/del_robot
2021-09-24 13:06:05,099 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/set_exc_robot_pose
2021-09-24 13:06:05,103 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/1/prepare_model
2021-09-24 13:06:05,107 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation initialized
2021-09-24 13:06:06,179 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Starting State Machines...
2021-09-24 13:07:23,923 [/ros_cle_sim] [hbp_nrp_back] [INFO]  State machine outcomes: 
[WARN] [1632481643.925060, 1.000000]: Could not process inbound connection: [/nrp_backend] is not a publisher of [/ros_cle_simulation/cle_error]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/ros_cle_simulation/1/lifecycle', 'cle_ros_msgs/SimulationLifecycleStateChange']]{'callerid': '/rosbridge_websocket', 'md5sum': 'a2eeba76fe73c5cc85d54c7b17af3ec8', 'message_definition': '# This message contains information on a error raised after a user update of some CLE related source code:\n# Transfer function, Smach script, PyNN script\nint32 severity # error severity according to the following constants\nint32 SEVERITY_WARNING=0  # The error might not affect the simulation at all\nint32 SEVERITY_ERROR=1    # The error will lead to a simulation failure if not resolved\nint32 SEVERITY_CRITICAL=2 # The error has lead to simulation failure\nint32 SEVERITY_MAJOR = 3 # The error has lead to stop the simulation and show a message in the frontend\n\nstring sourceType # e.g., "Transfer Function"\nstring SOURCE_TYPE_TRANSFER_FUNCTION=Transfer Function\nstring SOURCE_TYPE_STATE_MACHINE=State Machine\nstring errorType # e.g., "NoOrMultipleNames", "Compile", "Loading", "Runtime"\nstring message # description of the error, e.g., "IndentationError: unexpected indent"\n\nstring functionName # python def name of the function causing the error, empty if unavailable\n\n# the following fields are used when a python SyntaxError is raised, strings are left empty otherwise\nint32 lineNumber # line number of the error, -1 if unavailable \nint32 offset # python\'s SyntaxError offset, -1 if unavailable\nstring lineText # text of the line causing the error\nstring fileName # name of the file in which the error is raised (<string> if no such file exists)\n', 'tcp_nodelay': '0', 'topic': '/ros_cle_simulation/cle_error', 'type': 'cle_ros_msgs/CLEError'}
2021-09-24 13:08:45,139 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Could not generate textures, error occurred : Failed to communicate with the storage server, status code 500
2021-09-24 13:08:45,404 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Requesting simulation resources
2021-09-24 13:08:45,404 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation timeout initialized
2021-09-24 13:08:55,688 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/reset
2021-09-24 13:08:55,692 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/extend_timeout
2021-09-24 13:08:55,696 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/get_transfer_functions
2021-09-24 13:08:55,700 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/add_transfer_function
2021-09-24 13:08:55,703 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/activate_transfer_function
2021-09-24 13:08:55,707 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/edit_transfer_function
2021-09-24 13:08:55,711 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/convert_transfer_function_raw_to_structured
2021-09-24 13:08:55,715 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/delete_transfer_function
2021-09-24 13:08:55,718 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/get_brain
2021-09-24 13:08:55,722 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/set_brain
2021-09-24 13:08:55,726 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/get_populations
2021-09-24 13:08:55,730 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/set_populations
2021-09-24 13:08:55,733 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/get_CSV_recorders_files
2021-09-24 13:08:55,737 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/simulation_recorder
2021-09-24 13:08:55,741 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/get_robots
2021-09-24 13:08:55,744 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/add_robot
2021-09-24 13:08:55,748 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/del_robot
2021-09-24 13:08:55,751 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/set_exc_robot_pose
2021-09-24 13:08:55,755 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/2/prepare_model
2021-09-24 13:08:55,759 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation initialized
2021-09-24 13:08:56,822 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Starting State Machines...
2021-09-24 13:09:51,287 [/ros_cle_sim] [hbp_nrp_back] [INFO]  State machine outcomes: 
[WARN] [1632481791.288857, 1.000000]: Could not process inbound connection: [/nrp_backend] is not a publisher of [/ros_cle_simulation/cle_error]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/ros_cle_simulation/2/lifecycle', 'cle_ros_msgs/SimulationLifecycleStateChange']]{'callerid': '/rosbridge_websocket', 'md5sum': 'a2eeba76fe73c5cc85d54c7b17af3ec8', 'message_definition': '# This message contains information on a error raised after a user update of some CLE related source code:\n# Transfer function, Smach script, PyNN script\nint32 severity # error severity according to the following constants\nint32 SEVERITY_WARNING=0  # The error might not affect the simulation at all\nint32 SEVERITY_ERROR=1    # The error will lead to a simulation failure if not resolved\nint32 SEVERITY_CRITICAL=2 # The error has lead to simulation failure\nint32 SEVERITY_MAJOR = 3 # The error has lead to stop the simulation and show a message in the frontend\n\nstring sourceType # e.g., "Transfer Function"\nstring SOURCE_TYPE_TRANSFER_FUNCTION=Transfer Function\nstring SOURCE_TYPE_STATE_MACHINE=State Machine\nstring errorType # e.g., "NoOrMultipleNames", "Compile", "Loading", "Runtime"\nstring message # description of the error, e.g., "IndentationError: unexpected indent"\n\nstring functionName # python def name of the function causing the error, empty if unavailable\n\n# the following fields are used when a python SyntaxError is raised, strings are left empty otherwise\nint32 lineNumber # line number of the error, -1 if unavailable \nint32 offset # python\'s SyntaxError offset, -1 if unavailable\nstring lineText # text of the line causing the error\nstring fileName # name of the file in which the error is raised (<string> if no such file exists)\n', 'tcp_nodelay': '0', 'topic': '/ros_cle_simulation/cle_error', 'type': 'cle_ros_msgs/CLEError'}
2021-09-24 13:10:48,309 [Thread-5    ] [hbp_nrp_back] [INFO]  Start cleanup
2021-09-24 13:11:17,040 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Could not generate textures, error occurred : Failed to communicate with the storage server, status code 500
2021-09-24 13:11:17,223 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Requesting simulation resources
2021-09-24 13:11:17,223 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation timeout initialized
2021-09-24 13:11:25,481 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/reset
2021-09-24 13:11:25,485 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/extend_timeout
2021-09-24 13:11:25,489 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/get_transfer_functions
2021-09-24 13:11:25,493 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/add_transfer_function
2021-09-24 13:11:25,497 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/activate_transfer_function
2021-09-24 13:11:25,501 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/edit_transfer_function
2021-09-24 13:11:25,505 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/convert_transfer_function_raw_to_structured
2021-09-24 13:11:25,509 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/delete_transfer_function
2021-09-24 13:11:25,513 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/get_brain
2021-09-24 13:11:25,516 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/set_brain
2021-09-24 13:11:25,521 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/get_populations
2021-09-24 13:11:25,525 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/set_populations
2021-09-24 13:11:25,529 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/get_CSV_recorders_files
2021-09-24 13:11:25,534 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/simulation_recorder
2021-09-24 13:11:25,538 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/get_robots
2021-09-24 13:11:25,542 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/add_robot
2021-09-24 13:11:25,546 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/del_robot
2021-09-24 13:11:25,550 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/set_exc_robot_pose
2021-09-24 13:11:25,554 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/3/prepare_model
2021-09-24 13:11:25,558 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation initialized
2021-09-24 13:11:26,625 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Starting State Machines...
2021-09-24 13:12:06,594 [/ros_cle_sim] [hbp_nrp_back] [INFO]  State machine outcomes: 
[WARN] [1632481926.595434, 1.000000]: Could not process inbound connection: [/nrp_backend] is not a publisher of [/ros_cle_simulation/cle_error]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/ros_cle_simulation/3/lifecycle', 'cle_ros_msgs/SimulationLifecycleStateChange']]{'callerid': '/rosbridge_websocket', 'md5sum': 'a2eeba76fe73c5cc85d54c7b17af3ec8', 'message_definition': '# This message contains information on a error raised after a user update of some CLE related source code:\n# Transfer function, Smach script, PyNN script\nint32 severity # error severity according to the following constants\nint32 SEVERITY_WARNING=0  # The error might not affect the simulation at all\nint32 SEVERITY_ERROR=1    # The error will lead to a simulation failure if not resolved\nint32 SEVERITY_CRITICAL=2 # The error has lead to simulation failure\nint32 SEVERITY_MAJOR = 3 # The error has lead to stop the simulation and show a message in the frontend\n\nstring sourceType # e.g., "Transfer Function"\nstring SOURCE_TYPE_TRANSFER_FUNCTION=Transfer Function\nstring SOURCE_TYPE_STATE_MACHINE=State Machine\nstring errorType # e.g., "NoOrMultipleNames", "Compile", "Loading", "Runtime"\nstring message # description of the error, e.g., "IndentationError: unexpected indent"\n\nstring functionName # python def name of the function causing the error, empty if unavailable\n\n# the following fields are used when a python SyntaxError is raised, strings are left empty otherwise\nint32 lineNumber # line number of the error, -1 if unavailable \nint32 offset # python\'s SyntaxError offset, -1 if unavailable\nstring lineText # text of the line causing the error\nstring fileName # name of the file in which the error is raised (<string> if no such file exists)\n', 'tcp_nodelay': '0', 'topic': '/ros_cle_simulation/cle_error', 'type': 'cle_ros_msgs/CLEError'}
2021-09-24 13:13:28,594 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Could not generate textures, error occurred : Failed to communicate with the storage server, status code 500
2021-09-24 13:13:28,793 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Requesting simulation resources
2021-09-24 13:13:28,793 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation timeout initialized
2021-09-24 13:13:36,015 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/reset
2021-09-24 13:13:36,019 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/extend_timeout
2021-09-24 13:13:36,023 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/get_transfer_functions
2021-09-24 13:13:36,026 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/add_transfer_function
2021-09-24 13:13:36,030 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/activate_transfer_function
2021-09-24 13:13:36,034 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/edit_transfer_function
2021-09-24 13:13:36,038 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/convert_transfer_function_raw_to_structured
2021-09-24 13:13:36,042 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/delete_transfer_function
2021-09-24 13:13:36,045 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/get_brain
2021-09-24 13:13:36,049 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/set_brain
2021-09-24 13:13:36,053 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/get_populations
2021-09-24 13:13:36,057 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/set_populations
2021-09-24 13:13:36,061 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/get_CSV_recorders_files
2021-09-24 13:13:36,065 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/simulation_recorder
2021-09-24 13:13:36,070 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/get_robots
2021-09-24 13:13:36,074 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/add_robot
2021-09-24 13:13:36,078 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/del_robot
2021-09-24 13:13:36,081 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/set_exc_robot_pose
2021-09-24 13:13:36,085 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Connecting to ROS service /ros_cle_simulation/4/prepare_model
2021-09-24 13:13:36,089 [uWSGIWorker1] [hbp_nrp_back] [INFO]  simulation initialized
2021-09-24 13:13:37,150 [uWSGIWorker1] [hbp_nrp_back] [INFO]  Starting State Machines...
2021-09-24 13:14:03,158 [/ros_cle_sim] [hbp_nrp_back] [INFO]  State machine outcomes: 
[WARN] [1632482043.159543, 1.000000]: Could not process inbound connection: [/nrp_backend] is not a publisher of [/ros_cle_simulation/cle_error]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/ros_cle_simulation/4/lifecycle', 'cle_ros_msgs/SimulationLifecycleStateChange']]{'callerid': '/rosbridge_websocket', 'md5sum': 'a2eeba76fe73c5cc85d54c7b17af3ec8', 'message_definition': '# This message contains information on a error raised after a user update of some CLE related source code:\n# Transfer function, Smach script, PyNN script\nint32 severity # error severity according to the following constants\nint32 SEVERITY_WARNING=0  # The error might not affect the simulation at all\nint32 SEVERITY_ERROR=1    # The error will lead to a simulation failure if not resolved\nint32 SEVERITY_CRITICAL=2 # The error has lead to simulation failure\nint32 SEVERITY_MAJOR = 3 # The error has lead to stop the simulation and show a message in the frontend\n\nstring sourceType # e.g., "Transfer Function"\nstring SOURCE_TYPE_TRANSFER_FUNCTION=Transfer Function\nstring SOURCE_TYPE_STATE_MACHINE=State Machine\nstring errorType # e.g., "NoOrMultipleNames", "Compile", "Loading", "Runtime"\nstring message # description of the error, e.g., "IndentationError: unexpected indent"\n\nstring functionName # python def name of the function causing the error, empty if unavailable\n\n# the following fields are used when a python SyntaxError is raised, strings are left empty otherwise\nint32 lineNumber # line number of the error, -1 if unavailable \nint32 offset # python\'s SyntaxError offset, -1 if unavailable\nstring lineText # text of the line causing the error\nstring fileName # name of the file in which the error is raised (<string> if no such file exists)\n', 'tcp_nodelay': '0', 'topic': '/ros_cle_simulation/cle_error', 'type': 'cle_ros_msgs/CLEError'}
